infra/test/setup/versions.tf

```diff
@@ -19,7 +19,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.65"
+      version = "~> 5.4"
```

Comment:
**Suggestion:** Feel free to delete `infra/test/setup/versions.tf` so that our integration tests matches what our JSS users run as closely as possible. (We'll only have an issue here if/when — in the future — this file deviates from `infra/provider.tf`.)